src/main/java/com/github/manolo8/darkbot/backpage/GalaxyManager.java

```diff
@@ -0,0 +1,104 @@
+package com.github.manolo8.darkbot.backpage;
+
+import com.github.manolo8.darkbot.Main;
+import com.github.manolo8.darkbot.backpage.entities.galaxy.*;
+import org.dom4j.Document;
+import org.dom4j.Element;
+import org.dom4j.io.SAXReader;
+
+import java.net.HttpURLConnection;
+import java.util.Iterator;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class GalaxyManager {
+
+    private GalaxyInfo galaxyInfo;
+    private BackpageManager backpageManager;
+    private Main main;
+    private long lastGatesUpdate;
+
+    public GalaxyManager(Main main, BackpageManager backpageManager) {
+        this.main = main;
+        this.backpageManager = backpageManager;
+        this.galaxyInfo = new GalaxyInfo();
+    }
+
+    public GalaxyInfo getGalaxyInfo() {
+        return galaxyInfo;
+    }
+
+    public long lastGatesUpdate() {
+        return System.currentTimeMillis() - lastGatesUpdate;
+    }
+
+    /**
+     * @param gate       choose gate from GatesList to spin
+     * @param multiplier boolean to use a multiplier
+     * @param spinAmount amount of energy to spin {5, 10, 100}, set 0 to spin one energy
+     * @param minWait    minimum delay between requests
+     * @return response code from connection
+     */
+
```

Comment:
remove empty line, also now that the parameters are very obvious documentation probably not required